src/App.css

```diff
@@ -51,42 +51,34 @@
   font-size: 32px;
 }
 
-.MuiFab-root {
-  height: 18px !important;
+.MainGrid {
+  padding: 20px;
 }
 
-.MuiFab-primary {
-  border-radius: 0% !important;
```

Comment:
It's good that these have been replaced. Generally `!important` should be avoided wherever possible.